nodejs/week1/avg.js

```diff
@@ -0,0 +1,15 @@
+const inputValues = process.argv.slice(2);
+const numbersFromValues = inputValues.map((item) => Number(item));
```

Comment:
Very nice to extract these values into variables with explicative names. Way better than using `process.argv.length` all the time. Great! 👏